components/form/ColLayoutParam.cs

```diff
@@ -31,25 +31,25 @@ public class ColLayoutParam
 
         public OneOf<int, EmbeddedProperty> Xxl { get; set; }
 
-        public AntLableAlignType LableAlignType { get; set; } = AntLableAlignType.Right;
+        public AntLableAlignType LableAlign { get; set; } = AntLableAlignType.Right;
```

Comment:
typo `LabelAlign`

Reply:
Sorry for such a low-level error, fixed, please review